persister/whisper.go

```diff
@@ -3,6 +3,7 @@ package persister
 import (
 	"errors"
 	"fmt"
+	prom "github.com/prometheus/client_golang/prometheus"
```

Comment:
linter warning
> File is not `goimports`-ed (goimports)

Reply:
in general I think renaming import is frowned upon since it's adds extra complexity to remember not the package name but import name, maybe it's really needed in this case though, not sure

Reply:
I think it just want to separate import for named importing. But it's just easier to [include goimports to gofmt](https://pkg.go.dev/golang.org/x/tools/cmd/goimports).

Reply:
thanks, removed renaming and fixed linter